gatsby-config.js

```diff
@@ -1,4 +1,9 @@
+const activeEnv =
+  process.env.GATSBY_ACTIVE_ENV || process.env.NODE_ENV || "development"
```

Comment:
Here I was trying to connect to a .env.development file to hide API keys under the .env file but was getting a `can't resolve fs in node_modules` type of error when hooked up with require("dotenv").config({path: `.env.${activeEnv}``